www-dev/index.html

```diff
@@ -15,13 +15,22 @@
         
         function auth() {
             var user = prompt("Please enter your username, or leave blank to use without logging in");
+            var pass;
+            if(user)
```

Comment:
I haven't added a style guide yet, but I'd like to stick with the existing convention I've used.

When you have a "one-liner" if statement like here, just put it on the same line. So just do `if (user) pass = ...` without the line break.

Also, for if statements, code blocks, and operations please add a space
- `if(condition)` becomes `if (condition)`
- `if (condition){}` becomes `if (condition){}`
- `foo=bar` becomes `foo = bar`
-  `foobar = foo+bar` becomes `foobar = foo + bar`